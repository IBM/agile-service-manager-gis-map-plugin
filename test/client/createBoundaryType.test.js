// Import the module under test
import {createBoundaryType} from '../../src/scripts/client/createBoundaryType';

describe('createBoundaryType', function() {
    test('create', function() {        
        expect(createBoundaryType('myType')).toMatchSnapshot();
    });
});
