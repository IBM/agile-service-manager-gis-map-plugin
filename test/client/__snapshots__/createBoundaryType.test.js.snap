// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createBoundaryType create 1`] = `
Object {
  "boundaries": Object {},
  "boundaryType": "myType",
  "clusterGroup": NewClass {
    "_initHooksCalled": true,
    "_layers": Object {},
    "options": Object {},
  },
}
`;
